react/features/base/participants/reducer.ts

```diff
@@ -220,6 +225,16 @@ ReducerRegistry.register<IParticipantsState>('features/base/participants',
             }
         }
 
+        if (oldParticipant && newParticipant && !newParticipant.fakeParticipant
+            && !isScreenShareParticipant(newParticipant)) {
+            if (oldParticipant.e2eeEnabled !== newParticipant.e2eeEnabled) {
+                state.numberOfParticipantsDisabledE2EE += newParticipant.e2eeEnabled ? -1 : 1;
```

Comment:
Do you think that a counter is better than having a Set with the participant IDs that have e2ee and another one for those which support it?

Doing some set intesections sounds a bit easier to not mess up I think.

Reply:
I was thinking about it at first!

IMHO for now we are good with the counter. 

I couldn't think of much benefits with the Set. And we won't be using the values of the Set for now, we will be mainly checking  the size of it. In addition probably performance-wise the difference will be negligible but the counter should be with better performance. 

The only thing I can think of that will be better with the Set is that we won't be able to push the same ID twice. But anyway if we are trying to do this I think it is better to be able to see the problem and fix it which will be easier with the counter, the Set will just mask the problem.